### PR TITLE
Add logout command / Session storage in home / Auto select role

### PR DESCRIPTION
**Pull request by loopingz**
Problem Statement
-----------------
Once login you could not logout easily ( you need to remove by hand the session file )
The session file end up in the working folder which can leads to security issue
If there is only one role available it does not auto select it

Solution
--------
Moving the session to HOME directory
Adding a 'logout' command
Autoselect role if only one detected

### CODE REVIEW

**Comment by mraible** [comment]
I'd be happy to merge this, but I'd be doing it blindly. Can someone that uses this tool approve this PR?

**Comment by AlainODea** [comment]
@mraible I'll QA this and come back with a code review.

**Comment by AlainODea** [comment]
@loopingz can you build and add **out/oktaawscli.jar** to this PR?

```bash
mvn package
cp target/okta-aws-cli-*.jar out/oktaawscli.jar
```

I don't love this process for releases, but it's what we have ;)

File: src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java [hunk 1]
```diff
@@ -293,44 +308,55 @@ private AssumeRoleWithSAMLRequest chooseAwsRoleToAssume(String samlResponse) thr
...
-                    j = i;
+        String principalArn;
+        String roleArn;
+
+        if (roleIdpPairs.size() > 1) {
+            List<AccountOption> accountOptions = getAvailableRoles(samlResponse);
+
+            System.out.println("\nPlease choose the role you would like to assume: ");
+            System.out.println(roleIdpPairs.toString());
+            System.out.println(accountOptions.toString());
```
Comment (AlainODea) [inline src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java]: :art: can two outputs be removed? The menu is rendered by the loop below.

**Comment by loopingz** [comment]
We should build with travis and publish to a S3 bucket, it would be much better process that include huge binary in the repository, i dont mind setting up the travis but i m not official contributor of this repo, and Okta would have to provide us with a bucket and keys to access it

**Comment by mraible** [comment]
Since this is a Maven project, we could set it up so it publishes to Maven Central when Travis succeeds. Would that solve the problem?

**Comment by AlainODea** [comment]
I think given that @smashling provided the JAR with these changes that this PR should be merged and I suggest moving the Travis discussion to [Issue #72](https://github.com/oktadeveloper/okta-aws-cli-assume-role/issues/72)

**Comment by AlainODea** [comment]
Thank you for this contribution, @loopingz. This solves a couple of significant issues for my users.

**Comment by loopingz** [comment]
You are welcome the roles menu selection was a great contribution of yours too.
I think we should address the travis build and probably initate a good gh-pages for this project, i'll probably write a blog later but would be nice to have some public relation help from Okta

**Comment by mraible** [comment]
Hello @loopingz. I'm on the Developer Relations team at Okta. I'm happy to help however I can just let me know.

**Pull request merged**